padertorch/data/fragmenter.py

```diff
@@ -109,86 +119,145 @@ def fragment(key, x):
         return fragments
 
 
-
-def get_segment_offsets(num_samples, length, shift=None,
-                        offset_mode='start', num_segments=None):
+def get_segment_boundaries(num_samples, length, shift=None,
+                           mode='begin', choose_one=False, rng=np.random):
     """
```

Comment:
Add a description of this function at the beginning of the function. (What does this function do? best in one sentence)

Reply:
done